tests/acceptance/posts-test.js

```diff
@@ -1,95 +1,89 @@
 import { click, find, waitUntil, visit } from '@ember/test-helpers';
 import { module, test } from 'qunit';
 import { setupApplicationTest } from 'ember-qunit';
+import { getPageTitle } from '../helpers/get-page-title';
 
 module('Acceptance: title', function(hooks) {
   setupApplicationTest(hooks);
 
-  // Testem appends progress to the title...
-  // and there's no way to stop this at the moment
-  function title() {
-    let element = document.querySelector('head title');
-    return element && element.innerText.trim().replace(/^\(\d+\/\d+\)/, '');
-  }
-
   test('the default configuration works', async function (assert) {
     assert.expect(1);
     await visit('/posts');
 
-    assert.equal(title(), 'Posts | My App');
+    assert.equal(getPageTitle(window.document), 'Posts | My App');
```

Comment:
Can we default this param (instead of passing it all the time)?

Reply:
We can.